bokeh/models/callbacks.py

```diff
@@ -56,17 +64,10 @@ def from_py_func(cls, func):
     """)
 
     code = String(default="", help="""
-    A snippet of JavaScript/CoffeeScript code to execute in the browser. The
+    A snippet of JavaScript code to execute in the browser. The
```

Comment:
Would be appropriate to mention the JS string can be made with `CoffeeScript` class from etc.